.pre-commit-config.yaml

```diff
@@ -1,44 +1,48 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks.git
-    rev: v2.0.0
-    hooks:
-    -   id: trailing-whitespace
-        language_version: python3.6
-    -   id: end-of-file-fixer
-        language_version: python3.6
-    -   id: check-json
-        files: \.(jshintrc|json)$
-        language_version: python3.6
-    -   id: check-yaml
-        language_version: python3.6
-    -   id: debug-statements
-        language_version: python3.6
-    -   id: name-tests-test
-        files: ^tests/.+\.py$
-        language_version: python3.6
-    -   id: flake8
-        language_version: python3.6
-    -   id: requirements-txt-fixer
-        language_version: python3.6
-    -   id: check-added-large-files
-        language_version: python3.6
-    -   id: check-byte-order-marker
-        language_version: python3.6
-    -   id: fix-encoding-pragma
-        language_version: python3.6
-        args: [--remove]
--   repo: https://github.com/asottile/reorder_python_imports.git
-    rev: v1.3.3
-    hooks:
-    -   id: reorder-python-imports
-        language_version: python3.6
-        args: [
-            --remove-import, 'from __future__ import absolute_import',
-            --remove-import, 'from __future__ import print_function',
-            --remove-import, 'from __future__ import unicode_literals',
-        ]
--   repo: https://github.com/pre-commit/mirrors-autopep8
-    rev: v1.4.2
-    hooks:
-    -   id: autopep8
-        language_version: python3.6
+    - repo: https://github.com/pre-commit/pre-commit-hooks.git
+      rev: v2.0.0
+      hooks:
+          - id: trailing-whitespace
+            language_version: python3.8
```

Comment:
we can make this easier for the next upgrade by using the `default_language_version` feature of pre-commit (and this also has the side-effect of cleaning up/minimizing the  config)

Reply:
`default_language_version ` was giving an error but `python3` works